3.10/architecture-replication.md

```diff
@@ -28,62 +28,68 @@ Synchronous replication only works within an ArangoDB Cluster and is typically
-with a value greater than _1_ locks down a collection's leader shards for
-writing as soon as too many followers were lost.
+_leader_ and `r - 1` _followers_, where `r` denotes the **replication factor**.
+The replication factor is the total number of copies that are kept, that is, the
+leader and follower count combined. For example, with a replication factor of
+`3`, there is one _leader_ and `3 - 1 = 2` _followers_. You can control the
+number of _followers_ using the `replicationFactor` parameter whenever you
+create a _collection_, by setting a `replicationFactor` one higher than the
+desired number of followers. You can also adjust the value later.
+
+You cannot set a `replicationFactor` higher than the number of available
+DB-Servers by default. You can bypass the check when creating a collection by
+setting the `enforceReplicationFactor` option to `false`. You cannot bypass it
+when adjusting the replication factor later. Note that the replication factor
+is not decreased but remains the same during a DB-Server node outage.
+
+In addition to the replication factor, there is a **writeConcern** that
+specifies the minimum number of in-sync followers required for write operations.
+If you specify the `writeConcern` parameter with a value greater than `1`, the
+collection's leader shards are locked down for writing as soon as too few
+followers are available.
 
 Asynchronous replication
 ------------------------
 
-In ArangoDB any write operation is logged in the _write-ahead
-log_. 
+Any write operation in ArangoDB is logged in the _write-ahead log_.
 
-When using asynchronous replication _Followers_ (or _followers_)  
-connect to a _Leader_ (or _leader_) and apply locally all the events from
-the Leader log in the same order. As a result the _Followers_ (_followers_) 
-will have the same state of data as the _Leader_ (_leader_).
+When using asynchronous replication, _Followers_ connect to a _Leader_ and apply
+all the events from the Leader log in the same order locally. As a result, the
+_Followers_ end up with the same state of data as the _Leader_.
 
-_Followers_ (_followers_) are only eventually consistent with the _Leader_ (_leader_).
+_Followers_ are only eventually consistent with the _Leader_.
 
-Transactions are honored in replication, i.e. transactional write operations will 
+Transactions are honored in replication, i.e. transactional write operations 
 become visible on _Followers_ atomically.
 
-As all write operations will be logged to a Leader database's _write-ahead log_, the 
-replication in ArangoDB currently cannot be used for write-scaling. The main purposes 
-of the replication in current ArangoDB are to provide read-scalability and "hot backups" 
+As all write operations are logged to a Leader database's _write-ahead log_, the 
+replication in ArangoDB cannot be used for write-scaling. The main purposes 
+of the replication are to provide read-scalability and "hot backups"
```

Comment:
The usage of the term "hot backups" here is misleading, since we have used this for something else, since this was written. How about "hot standby servers" or something like this?